R/Reactant/platform_augmentation.jl

```diff
@@ -93,8 +93,10 @@ function augment_platform!(platform::Platform)
             Libdl.dlclose(handle)
 
             if cuda_version_tag == "none" && current_cuda_version isa VersionNumber
-                if v"12.1" <= current_cuda_version < v"12.6"
+                if v"12.1" <= current_cuda_version < v"12.3"
                     cuda_version_tag = "12.1"
+                elseif v"12.3" <= current_cuda_version < v"12.6"
```

Comment:
I was kind of suggesting to do

if VERSION > v12.3
  cuda_version_tag = "12.6"
elseif VERSION > v12
  cuda_version_tag = "12.1"
else..

Reply:
> if VERSION > v12.3
> cuda_version_tag = "12.6"

Wait, I find that that confusing. Are you suggesting that if system has CUDA later than 12.3 (so 12.4+) then we use our 12.6 build? I'm not sure that's going to work, but maybe I'm misunderstanding something.

Reply:
it's possible yeah, but as we saw elsewhere the insides of XLA actually does check dynamically the cuda version. However, if ver >= 12.3then it links against something that makes loading impossible. So we should put something at that boundary point

Reply:
Billy approved offline